website/blog/2023-05-07-databend-weekly-92.md

```diff
@@ -0,0 +1,112 @@
+
+Databend now supports column aliases in WHERE clause. Note that if the alias name is the same as the column name, the column name should be used first.
+
+```sql
+> select number * 2 as number from numbers(3) where (number + 1) % 3 = 0;
+
+┌────────┐
+│ number │
+│ UInt64 │
+├────────┤
+│      4 │
+└────────┘
+```
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [PR #11272 | feat: support alias in where clause](https://github.com/datafuselabs/databend/pull/11272)
+
+### `databend-metactl` Is Included in the Databend Release
+
+`databend-metactl` is a command-line tool that allows users to manage the Databend Meta Service cluster. It can be used to back up and restore the metadata.
+
+Now, databend-metactl will be released together with Databend and no longer requires building it manually.
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [PR #11291 | feat: add bitmap data type](https://github.com/datafuselabs/databend/pull/11291)
+- [Docs | Back Up and Restore Databend Meta Service Cluster](https://databend.rs/doc/deploy/metasrv/metasrv-backup-restore)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Troubleshooting `databend-meta` Connection Issues with Invalid Semver Error
+
+When compiling and running Databend, you may encounter connection issues with databend-meta and receive an error message containing `Invalid semver`. This is due to Databend's protocol compatibility check using semantic versions.
+
+To resolve this issue, you can fetch the latest tags from the official Databend repository using the command `git fetch https://github.com/datafuselabs/databend.git --tags`. This ensures that your project is using the latest version of databend-meta and passes the version check.
+
+## Highlights
+
+Here are some noteworthy items recorded here, perhaps you can find something that interests you.
+
+- Optimizer now supports constant folding. If interested, please check [PR #11216](https://github.com/datafuselabs/databend/pull/11216).
+- Learn how to [Transform Data During Load](https://databend.rs/doc/load-data/data-load-transform) through three simple tutorials.
+- Added bitmap functions: `bitmap_count` and `build_bitmap`.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Introduce Timeout Mechanism to Control the Query Optimization Time
+
+Optimizing complex queries with numerous joins can be time-consuming. To avoid exhaustive search of the entire query plan space, we can set a timeout for optimization. 
+
+The timeout need not be based on wall time, but rather on a logical time, such as the number of applied transform rules. Once the timeout is reached, instead of generating new transform rules, a greedy search can be performed.
```

Comment:
The timeout can be based on a logical time, such as the number of applied transform rules, rather than wall time. Once the timeout is reached, a greedy search can be performed instead of generating new transform rules.